tests/EmbeddedDSOs/EmbeddedDSO-UnitTests/BuildTests.cs

```diff
@@ -0,0 +1,299 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Xml;
+using System.Xml.XPath;
+
+using Microsoft.Build.Framework;
+using NUnit.Framework;
+using Xamarin.ProjectTools;
+using Xamarin.Tools.Zip;
+
+using XABuildPaths = global::Xamarin.Android.Build.Paths;
+
+namespace EmbeddedDSOUnitTests
+{
+	sealed class LocalBuilder : Builder
```

Comment:
I wonder if these unit tests should be integrated with/moved into `src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests`.

Reply:
I kept them separate because the Android project used by those tests is also a standalone test in its own right (with instrumentation and such)